beacon-chain/core/peerdas/util.go

```diff
@@ -0,0 +1,49 @@
+package peerdas
+
+import (
+	"github.com/prysmaticlabs/prysm/v5/beacon-chain/blockchain/kzg"
+	"github.com/prysmaticlabs/prysm/v5/config/params"
+	"github.com/prysmaticlabs/prysm/v5/consensus-types/interfaces"
+	ethpb "github.com/prysmaticlabs/prysm/v5/proto/prysm/v1alpha1"
+	"github.com/prysmaticlabs/prysm/v5/runtime/version"
+)
+
+// Helper function to unblind data column sidecars from a block and a blobs bundle v2.
```

Comment:
Godoc of public function should start with the name of the public function itself.